app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,140 @@
+would be available for application registration.
+
+## Multiple runtime groups
+{:.badge .enterprise}
+
+In addition to the default runtime group, you can also configure two additional
+custom groups. Splitting runtime instances and configuration into multiple
+runtime groups provides multiple specialized environments in one account.
+
+Some common use cases for using multiple runtime groups include:
+
+* **Environment separation:** Split environments based on their purpose, such as
+development, staging, and production.
+* **Region separation:** Dedicate each runtime group to a region or group of
+regions. Spin up runtime instances in those regions for each runtime group.
+* **Team separation:** Dedicate each runtime group to a different team and share
+resources based on team purpose.
+
+[Set up and manage runtime groups &rarr;](/konnect/configure/runtime-manager/runtime-groups/manage)
+
+## Administration
+
+Access to each runtime group is configurable on a team-by-team basis using
+entity-specific permissions.
+
+_Q: Do users default to the default runtime group?
+Q: What happens to users when a custom group is deleted?_
+
+[Administer teams &rarr;](/link)
+
+## Global configuration
+
+For each runtime group, you can spin up runtime instances and configure global
+options for the following {{site.base_gateway}} objects:
+* Consumers
+* Plugins
+* Upstreams
+* Certificates
+* SNIs
+
+A **global** object is a set of configurations that apply to, or can be used
+by, all objects in a runtime group. For example, if you set up a Proxy Caching
+plugin in the default runtime group and set it to `Global`,
+the plugin configuration will apply to all Service versions in the group.
+
+When there are multiple runtime groups, configuration for any of the global
+objects only applies to the runtime group that it was created in. Consumers and
+their authentication mechanisms don't carry over to other runtime groups.
+
+[Manage global entities &rarr;](/konnect/configure/runtime-manager/manage-entities)
+
+## References
+
+### Runtime groups table
+
+The following describes the content of the table found on the [Runtime Manager overview](https://konnect.konghq.com/runtime-manager/).
+
+Column | Description
+-------|-------------
+Name | The name of the runtime group. This group name must be unique in the organization.
+ID | A unique ID for the group. This ID is automatically generated when a group is created.
+Type | Types of runtimes that this group contains. Currently, the only supported runtime type is `Kong Gateway`.
+Active instances | The number of runtime instances in this group that are `Connected`, `Normal`, or `In Sync`.
+Labels | A list of labels on the runtime group in the form of key-value pairs.
+
+<!--State | The status of the runtime group. Can be one of: <br> &#8226; `initializing` - The group is currently being provisioned. <br> &#8226; `deployed` - The group has been provisioned and is available for use.-->
```

Comment:
On the mockups, I saw that the `State` column wasn't going to be used. Is this still the case?

Reply:
The columns in these tables still heavily depend on the new Control Plane that the Koko team is building.
The only thing I am certain about is that the `State` meaning in Helium is different from that of pre-Helium.

Reply:
Since the Koko CP is "multi-tenant", it is always in a "deployed" state so there isn't any initializing status to report on. If a new RG is created, it's still in the same CP (koko).